src/multivariate/mvnormal.jl

```diff
@@ -271,6 +278,18 @@ function _rand!(rng::AbstractRNG, d::MvNormal, x::AbstractVector)
     add!(unwhiten!(d.Σ, x), d.μ)
 end
 
+### Affine transformations
+
++(c::AbstractVector, d::MvNormal) = MvNormal(d.μ .+ c, d.Σ)
+
+@inline +(d::MvNormal, c::AbstractVector) = c + d
```

Comment:
Is the inline needed here?
```julia
@inline +(d::MvNormal, c::AbstractVector) = c + d
```

Reply:
I just added it because I was trying to follow the coding style of the package, which uses it pretty liberally. I don't think it is harmful.

Reply:
indeed not harmful. I removed a few of them, which were added I believe in a time and age when the compiler was less smart

Reply:
Also, one thing that I find harder to read: could we have `+(d, c)` be the "main" function and `+(c, d) = +(d, c)`? Having the "important argument first is easier to parse IMO

Reply:
bump

Reply:
thanks for the bump, done

Reply:
thanks, I think it's good to merge once the CI passes